apps/crn-server/src/data-providers/contentful/team.data-provider.ts

```diff
@@ -125,10 +148,7 @@ export class TeamContentfulDataProvider implements TeamDataProvider {
         }
       : {};
 
-    const activeQuery =
```

Comment:
This change standardizes the way `filters` work in this data provider to work in a similar way to other `*DataProviders`. The filters work as an array of strings, which will come handy in next sprints when we need to start filtering teams by different `Research Themes` or `Resource Types`.